2_Numpy_and_co.ipynb

```diff
@@ -79,7 +79,7 @@
     }
    ],
    "source": [
-    "x = math.cos(2 * math.pi)\n",
+    "x = math.cos(2*math.pi)\n",
```

Comment:
I'm not sure about this one, it may fall under E226 of e.g. [pycodestyle](https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes). Pep8 is rather ambiguous regarding this but I would advise keeping spaces.